apps/condo/domains/miniapp/schema/SendB2CAppPushMessageService.js

```diff
@@ -135,25 +139,29 @@ const SendB2CAppPushMessageService = new GQLCustomSchema('SendB2CAppPushMessageS
                 if (!residentExisted) throw new GQLError(ERRORS.RESIDENT_NOT_FOUND, context)
 
                 let B2CAppName = 'Debug app'
+                let appSettings = {}
 
                 // App requested to send notification to is not a DEBUG one
                 if (app.id !== DEBUG_APP_ID) {
                     const appExisted = await B2CApp.getOne(context, { id: app.id, deletedAt: null })
 
                     if (!appExisted) throw new GQLError(ERRORS.APP_NOT_FOUND, context)
 
-                    B2CAppName = appExisted.name
                     const where = { app: { id: app.id }, type, deletedAt: null }
-                    const appInBlackList = await MessageAppBlackList.getOne(context, where)
+                    appSettings = await B2CAppMessageSetting.getOne(context, where)
 
-                    if (appInBlackList) throw new GQLError(ERRORS.APP_IN_BLACK_LIST, context)
+                    if (get(appSettings, 'isBlacklisted') === true) throw new GQLError(ERRORS.APP_IN_BLACK_LIST, context)
+                    B2CAppName = appExisted.name
                 }
 
                 const searchKey = `${type}_${app.id}_${user.id}`
                 const ttl = CACHE_TTL[type] || CACHE_TTL['DEFAULT']
 
-                await redisGuard.checkLock(searchKey, SERVICE_NAME, context)
-                await redisGuard.lock(searchKey, SERVICE_NAME, ttl)
+                await redisGuard.checkCustomLimitCounters(
+                    `${SERVICE_NAME}_${searchKey}`,
```

Comment:
If I'm correct, previously we use `some-app-search-key` patter to name something inside redis guards, isn't it?

Reply:
I didn't know that, I'll fix the code